python/sidecar/typing.py

```diff
@@ -206,6 +206,30 @@ class SearchResponse(RefStudioModel):
     results: list[S2SearchResult]
 
 
+class FileEntryBase(RefStudioModel):
+    name: str
+    path: str
+
+
+class FileEntry(FileEntryBase):
+    file_extension: str
+
+
+class FolderEntry(FileEntryBase):
+    children: list[FileEntry]
+
+
+class ProjectDetails(RefStudioModel):
```

Comment:
nit: we usually call the API output types `....Response`

Reply:
We _usually_ do, but we do not always. I'm finding repeatedly using `Response` to be redundant in my cases. If a function returns a type, it is by definition a "response."

Reply:
You are right, we use sometimes.

I find it useful to have a naming convention for the API output types in order to distinguish them to internal types. That also help the client code to use the correct type.